docs/2014/database-engine/implementing-lob-columns-in-a-memory-optimized-table.md

```diff
@@ -69,7 +69,7 @@ where BlobId = @BlobId
 order by SegmentationId  
   
 select @Blob  
-go</code></pre>  
+go```
```

Comment:
Please put closing tick marks on its own line. Thank you!

Reply:
I made this fix.